public_dropin_gpu_environments/python311_genai/dr_requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyarrow>=0.14.1,<=14.0.1
+datarobot-drum==1.10.21
```

Comment:
Don't you want to bump DRUM to the latest?

Reply:
I have no idea. This is copy of `python311_genai` with min changes to support GPU. `python311_genai` currently marked as owned by `custom-models`.